client/components/Team.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default function Team() {
+    return (
+        <div className="container">
+            <section id="aboutSection">
+                <div id="aboutDiv">
+                    <h1>OUR STORY</h1>
+                    <p>
+                        <strong style={{ color: '#2785d7ca' }}>GraphQL GATEWAY</strong> tool seriers
```

Comment:
See drive for suggestions on updated wording.

Reply:
I agree, the wording at the bottom of the google doc is crisp.

Reply:
Done.